ui/app/components/app/transaction-list-item/index.scss

```diff
@@ -14,17 +14,12 @@
     padding: 16px 20px;
     display: grid;
     grid-template-columns: 45px 1fr 1fr 1fr;
-    grid-template-areas:
-      "identicon action status primary-amount"
-      "identicon nonce status secondary-amount";
+    grid-template-areas: "identicon action-and-nonce status amounts";
 
     @media screen and (max-width: $break-small) {
       padding: .5rem 1rem;
-      grid-template-columns: 45px 5fr 3fr;
-      grid-template-areas:
-        "nonce nonce nonce"
-        "identicon action primary-amount"
-        "identicon status secondary-amount";
+      grid-template-columns: 45px 145px 3fr;
```

Comment:
I noticed that you're still omitting the fourth column here. Is there any reason for that? I'm not even sure what behavior to expect in this case, e.g. whether it defaults to `1fr` or something else. Certainly if you _don't_ specify something for the fourth column, using `3fr` seems rather pointless ("3 parts" of the remaining space, when those are the only parts)

Setting the second column as `145px` seems reasonable though - I can see why you did that. I do wonder whether that's long enough to fit any of the messages that might appear there though 🤔. I'm not sure where to find a full list of those, particularly the translated versions.

Reply:
oh right, the fourth column...

Reply:
Yeah, it defaults to `1fr` Just pushed a commit which makes that explicit

Fitting long messages is an existing problem on that screen, with or without that change. Which should be addressed through design, but probably not in this issue?

Reply:
Unfortunately there is still a regression here:

Current develop branch:
![develop](https://user-images.githubusercontent.com/2459287/67692901-9b1c5980-f97f-11e9-80f6-3342df07a605.png)

This branch:
![clipped](https://user-images.githubusercontent.com/2459287/67692914-9fe10d80-f97f-11e9-8f14-169b993a1716.png)

The only status long enough to exceed the `145px` is "Contract Deployment"; it looks like it needs at least `153px` to fit without being truncated.

Though more importantly, this has brought my attention to a more substantial change; that it _was_ three columns before, and is now four. The status was placed below the transaction category, not alongside it.

I would suggest restoring the third column, and leaving the status below the transaction category. Then there would be ample room to accommodate not only "Contract Deployment", but any translated category names that might be substantially longer.